source/includes/administration/_usage.md

```diff
@@ -54,6 +54,8 @@ Attributes | &nbsp;
 `usageType`<br/>*string* | Usage type of the record.
 `secondaryType`<br/>*string* | Secondary type of the record.
 
+*Note*: The utilityUsage reported by this API is the total usage recorded for the specified period. The resource commitment is not taken into account for this endpoint but is outlined in the endpoint `/top_level/organizations/{org_id}` below. 
```

Comment:
2nd word is not formatted correctly 'utilityUsage.

Reply:
It's supposed to correspond to the attribute returned by the API, 6 lines above. You could put some code syntax to make it more obvious: `utilityUsage`.